ERROR - 2024-09-26 16:09:03 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@12345', 'username,passwo...', Array, NULL, 'password')
#8 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@12345', Array, Array)
#9 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#13 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@12345', 'username,passwo...', Array, NULL, 'password')
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@12345', Array, Array)
#8 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-26 16:09:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `User`
WHERE `UserName` = :UserName:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@12345', 'username,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@12345', [...], [...])
 8 SYSTEMPATH\Controller.php(141): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\User.php(29): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\User->login()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:09:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxs12345', 'username,passwo...', Array, NULL, 'password')
#8 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxs12345', Array, Array)
#9 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#13 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxs12345', 'username,passwo...', Array, NULL, 'password')
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxs12345', Array, Array)
#8 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-26 16:09:31 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `User`
WHERE `UserName` = :UserName:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxs12345', 'username,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxs12345', [...], [...])
 8 SYSTEMPATH\Controller.php(141): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\User.php(29): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\User->login()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:09:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#9 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-09-26 16:09:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `announcement`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Web->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:09:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#9 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-09-26 16:09:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `announcement`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Web->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:09:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#9 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\IPMS\app\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\Web->index()
#7 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
#8 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-09-26 16:09:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `announcement`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Web.php(60): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Web->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Web))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:10:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw1234', 'username,passwo...', Array, NULL, 'password')
#8 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw1234', Array, Array)
#9 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#13 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw1234', 'username,passwo...', Array, NULL, 'password')
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw1234', Array, Array)
#8 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-26 16:10:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `User`
WHERE `UserName` = :UserName:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw1234', 'username,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw1234', [...], [...])
 8 SYSTEMPATH\Controller.php(141): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\User.php(29): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\User->login()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-09-26 16:10:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ipmsm', 3306, '', 0)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@1234', 'username,passwo...', Array, NULL, 'password')
#8 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@1234', Array, Array)
#9 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#13 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\IPMS\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\IPMS\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\IPMS\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\IPMS\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\IPMS\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\IPMS\app\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@1234', 'username,passwo...', Array, NULL, 'password')
#7 C:\xampp\htdocs\IPMS\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@1234', Array, Array)
#8 C:\xampp\htdocs\IPMS\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\IPMS\app\Controllers\User.php(29): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(932): App\Controllers\User->login()
#11 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\IPMS\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\IPMS\index.php(67): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-26 16:10:25 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `User`
WHERE `UserName` = :UserName:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\validation\UserRules.php(13): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(278): App\Validation\UserRules->validateUser('Qazxsw@1234', 'username,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('password', 'Password', 'Qazxsw@1234', [...], [...])
 8 SYSTEMPATH\Controller.php(141): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\User.php(29): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\User->login()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
